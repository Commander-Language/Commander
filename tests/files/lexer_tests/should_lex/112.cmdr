//Should lex continue statement
continue;