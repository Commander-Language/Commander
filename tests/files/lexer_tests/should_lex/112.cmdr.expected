BREAK 'break' 1:1
SEMICOLON ';' 1:9
END_OF_FILE 1:10